app/src/main/java/de/westnordost/streetcomplete/quests/playground_access/AddPlaygroundAccess.java

```diff
@@ -0,0 +1,46 @@
+package de.westnordost.streetcomplete.quests.playground_access;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.parking_access.AddParkingAccessForm;
+
+public class AddPlaygroundAccess extends SimpleOverpassQuestType
+{
+	@Inject public AddPlaygroundAccess(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "nodes, ways, relations with leisure=playground and (!access or access=unknown)";
+	}
+
+	public AbstractQuestAnswerFragment createForm()
+	{
+		return new YesNoQuestAnswerFragment();
+	}
+
+	public void applyAnswerTo(Bundle answer, StringMapChangesBuilder changes)
+	{
+		if(answer.getBoolean(YesNoQuestAnswerFragment.ANSWER)){
+			changes.add("access", "yes");
+		} else {
+			changes.add("access", "private");
+		}
+	}
+	@Override public String getCommitMessage() { return "Add whatever playground is public"; }
```

Comment:
Should not this be "Add **whether** playground is public"?

Reply:
I simplified the text here